Header header
Position pos
LaserPoint[] scan
float32 maxDist